am-lib/src/test/java/uk/gov/hmcts/reform/amlib/AccessManagementServiceValidationTest.java

```diff
@@ -57,9 +60,11 @@ void revokeResourceAccessMethodShouldRejectInvalidArguments(ExplicitAccessMetada
 
     @ParameterizedTest
     @ArgumentsSource(InvalidArgumentsProvider.class)
-    void filterResourceMethodShouldRejectInvalidArguments(String userId, Set<String> userRoles, Resource resource) {
+    void filterResourceMethodShouldRejectInvalidArguments(
+        String userId, Set<String> userRoles, Resource resource,
+        Map<JsonPointer, SecurityClassification> attributeMap) {
```

Comment:
rename attributeMap to attributeSecurityClassifications

Reply:
done